extensions/vertx-http/deployment-spi/src/main/java/io/quarkus/vertx/http/deployment/spi/GeneratedStaticResourcesBuildItem.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.vertx.http.deployment.spi;
+
+import java.util.Collections;
+import java.util.Objects;
+import java.util.Set;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+/**
+ * This build items aims to be a collection of all {@link GeneratedStaticResourceBuildItem} produced by extensions.
+ */
+public final class GeneratedStaticResourcesBuildItem extends SimpleBuildItem {
```

Comment:
I think this should be moved to deployment as it's not meant to be used externally.